tests/storage/datavolume.go

```diff
@@ -73,6 +75,67 @@ var _ = SIGDescribe("DataVolume Integration", func() {
 		}
 	})
 
+	Context("[rook-ceph]PVC expansion", func() {
+		table.DescribeTable("PVC expansion is detected by VM and can be fully used", func(volumeMode k8sv1.PersistentVolumeMode) {
+			checks.SkipTestIfNoFeatureGate(virtconfig.ExpandDisksGate)
+			dataVolume := tests.NewRandomDataVolumeWithHttpImport(tests.GetUrl(tests.CirrosHttpUrl), util.NamespaceTestDefault, k8sv1.ReadWriteOnce)
+			sc, exists := tests.GetCephStorageClass()
+			if !exists {
+				Skip("Skip OCS tests when Ceph is not present")
+			}
+			dataVolume.Spec.PVC.StorageClassName = &sc
+			dataVolume.Spec.PVC.VolumeMode = &volumeMode
+
+			vmi := tests.NewRandomVMIWithDataVolume(dataVolume.Name)
+			tests.AddUserData(vmi, "cloud-init", "#!/bin/bash\necho 'hello'\n")
+
+			_, err := virtClient.CdiClient().CdiV1beta1().DataVolumes(dataVolume.Namespace).Create(context.Background(), dataVolume, metav1.CreateOptions{})
+			Expect(err).ToNot(HaveOccurred())
+
+			vmi = tests.RunVMIAndExpectLaunchWithDataVolume(vmi, dataVolume, 500)
+
+			By("Expecting the VirtualMachineInstance console")
+			Expect(console.LoginToCirros(vmi)).To(Succeed())
+
+			pvc, err := virtClient.CoreV1().PersistentVolumeClaims(util.NamespaceTestDefault).Get(context.Background(), dataVolume.Name, metav1.GetOptions{})
+			Expect(err).ToNot(HaveOccurred())
+
+			By("Expanding PVC")
+			pvc.Spec.Resources.Requests[k8sv1.ResourceStorage] = resource.MustParse("2Gi")
+			_, err = virtClient.CoreV1().PersistentVolumeClaims(util.NamespaceTestDefault).Update(context.Background(), pvc, metav1.UpdateOptions{})
+			Expect(err).ToNot(HaveOccurred())
+
+			By("Waiting for notification about size change")
+			Eventually(func() error {
+				err := console.SafeExpectBatch(vmi, []expect.Batcher{
+					&expect.BSnd{S: "\n"},
+					&expect.BExp{R: console.PromptExpression},
```

Comment:
does online expansion work for FS pvcs as well as block? I thought only block device resize occurred online.  Am i mistaken?

Reply:
It works for FS PVCs as well